tests/flow/promises/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -583,15 +576,12 @@ Promise
   });
 
 // TODO: resolvedPromise<T> -> catch() -> then():T
-Promise
-  .resolve(0)
-  .catch(function(err) {})
-  .then(function(num) {
-    var a: number = num;
+Promise.resolve(0).catch(function(err) {}).then(function(num) {
```

Comment:
If only the last has a break, and it fits in one line, it's going to flow as you suggested.